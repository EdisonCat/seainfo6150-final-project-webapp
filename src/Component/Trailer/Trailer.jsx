import React from "react";
import styles from "./Trailer.module.css";
import {useHistory} from "react-router-dom";
export default function Trailer() {
    const history = useHistory();
    return (
        <div className={styles.container}>
            <iframe width="807" height="454" src={history.location.state ? history.location.state.url : "/"} title="YouTube video player" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
        </div>
    );
}