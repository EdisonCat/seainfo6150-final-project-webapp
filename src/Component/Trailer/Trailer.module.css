.container {
    display: flex;
    background: #333;
    min-height: 900px;
    justify-content: center;
    border-top: 1px solid gray;
    padding: 50px;
}
@media screen and (max-width: 800px) {
    iframe {
        width: 100%;
    }
}