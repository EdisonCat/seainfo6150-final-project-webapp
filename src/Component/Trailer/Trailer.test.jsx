import React from "react";
import Trailer from "./Trailer";
it("rendered Trailer correctly", () => {
	// eslint-disable-next-line no-undef
    const { container } = render(<Trailer />);
    expect(container).toMatchSnapshot();
});