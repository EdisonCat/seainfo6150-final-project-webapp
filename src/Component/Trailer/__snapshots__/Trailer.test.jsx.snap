// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendered Trailer correctly 1`] = `
<div>
  <div
    class="container"
  >
    <iframe
      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
      allowfullscreen=""
      frameborder="0"
      height="454"
      src="/"
      title="YouTube video player"
      width="807"
    />
  </div>
</div>
`;
